actions/build_fuzzers/action.yml

```diff
@@ -53,7 +53,7 @@ inputs:
     required: false
 runs:
   using: 'docker'
-  image: 'docker://gcr.io/oss-fuzz-base/clusterfuzzlite-build-fuzzers:v1'
+  image: 'docker://gcr.io/oss-fuzz-base/clusterfuzzlite-build-fuzzers:ubuntu-24-04-v1'
```

Comment:
FWIW I'm not sure how safe it is to change it here. As far as I can remember the `v1` tag used by projects using CFLite was updated automatically and I don't think it should start pointing at these versions automatically (because it can just break stuff that doesn't work with Ubuntu 24.04 yet. For example avahi fails under MSan on Ubuntu 24.04 (https://github.com/avahi/avahi/issues/787) so I'd keep Ubuntu 20.04 there for now).

(If it doesn't affect the v1 tag and another tag is going to be added instead it should be fine of course)

Reply:
I'm also worried about this, but your explanation makes sense.

Reply:
Agreed. It makes sense. I guess I thought that it would start pulling ubuntu-24-04 left and right but it isn't the case. As long as ubuntu-20-04 is used by default it should be fine then.